src/main/kotlin/com/team4099/robot2023/config/constants/Constants.kt

```diff
@@ -65,4 +65,9 @@ object Constants {
   object Alert {
     val TABS = arrayOf("Pre-match", "In-match")
   }
+
+  object Led {
```

Comment:
This is small enough that it is probably fine but ideally we should avoid unrelated changes going in PRs -- it makes it harder to review and introduces a higher likelihood that stuff gets merged without it working